src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsEmailPreferences/SettingsEditSettingsEmailPreferences.tsx

```diff
@@ -1,69 +1,36 @@
 import * as React from "react"
-import { Flex, Select, Text, useToasts } from "@artsy/palette"
-import { useSystemContext } from "v2/System/SystemContext"
+import { 
+  Button, 
+  Flex, 
+  Text 
+} from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { SettingsEditSettingsEmailPreferences_me } from "v2/__generated__/SettingsEditSettingsEmailPreferences_me.graphql"
-import { UpdateUserEmailPreferencesMutation } from "v2/Components/UserSettings/UserEmailPreferences/UserEmailPreferencesMutation"
-
-const fallbackFrequency = "weekly"
-
-const options = [
-  { text: "None", value: "none" },
-  { text: "Daily", value: "daily" },
-  { text: "Weekly", value: "weekly" },
-  { text: "Alerts Only", value: "alerts_only" },
-]
-
+import { RouterLink } from "v2/System/Router/RouterLink"
 interface SettingEditSettingsEmailPreferencesProps {
   me: SettingsEditSettingsEmailPreferences_me
 }
 
 export const SettingsEditSettingsEmailPreferences: React.FC<SettingEditSettingsEmailPreferencesProps> = ({
   me,
 }) => {
-  const { relayEnvironment } = useSystemContext()
-  const emailFrequency = me?.emailFrequency || fallbackFrequency
-
-  const { sendToast } = useToasts()
-
-  const handleSelect = async newEmailFrequency => {
-    const variables = { emailFrequency: newEmailFrequency }
-    try{
-      await UpdateUserEmailPreferencesMutation(
-        // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-        relayEnvironment,
-        variables,
-        me?.id
-      )
-      sendToast({
-        variant: "success",
-        message: "Preference Updated",
-      })
-    } catch (err) {
-      sendToast({ variant: "error", message: err.message })
-    }
-  }
 
   return (
     <>
       <Text color="black100" variant="lg" mb={4}>
         Email Preferences
       </Text>
-      <Text color="black60" variant="sm">
-        Receive emails from Artsy with auctions, articles, curated collections,
-        and new works by artists you follow
-      </Text>
       <Flex mt="2" justifyContent="space-between">
-        <Text color="black100" variant="sm">
-          Frequency
+        <Text color="black60" variant="sm">
+          Receive emails from Artsy with auctions, articles, curated collections,
+          and new works by artists you follow
         </Text>
         <Flex alignItems="center">
-          <Select
-            variant="inline"
-            onSelect={handleSelect}
-            options={options}
-            selected={emailFrequency}
-          />
+          <RouterLink to="/preferences2">
+            <Button>
```

Comment:
This wouldn't work correctly as there'd be two tab stops (the link then the button, the button of course isn't doing anything).

The correct form would be to cast the button as a link: `Button as={RouterLink}`

Reply:
It'd work but not correctly. There's the accessibility issue of having a button that doesn't do anything when navigated via keyboard. Clicks would work since it'd propagate to the link.

Regarding the type error, it's OK to just `@ts-ignore` that line. This is sort of a long standing issue that's difficult to resolve with the `as` prop.

Reply:
Ah got it. Updated.